### ISSUE TITLE
Autocomplete attribute missing

### ISSUE THREAD

**Issue by emileblume** [einpraegsam/powermail#789]
WCAG regulations require autocomplete attributes for certain input values like 'given-name', 'family-name' and 'email'. 
See: https://www.w3.org/TR/WCAG21/#input-purposes

These options aren't available in Powermail.

**Comment by risingdam** [comment]
+1 for me We need this to pass WCAG 2.1 AA.

**Comment by ElectricMud** [comment]
Any update on when this option will be included? Seems like a no-brainer to me since it's mandatory to comply to WCAG

**Comment by einpraegsam** [comment]
If anyone is interested in supporting this feature, please contact @sbusemann

**Comment by ElectricMud** [comment]
It seems this was already picked up: https://github.com/einpraegsam/powermail/pull/620 but just needs to be included. 
@einpraegsam When can we expect this to happen?

**Comment by proudnerds-typo3** [comment]
@einpraegsam Will this option be finally available for TYPO3 v13?
